test/node/schemas/vis-state-schema-test.js

```diff
@@ -208,7 +208,7 @@ test('#visStateSchema -> v1 -> save load interaction -> tooltip format', t => {
     tooltip: {
       enabled: true,
       fieldsToShow: {
-        [testCsvDataId]: [{name: 'gps_data.utc_timestamp', format: 'LL'}],
+        [testCsvDataId]: [{name: 'gps_data.utc_timestamp', format: 'DATE_LL'}],
```

Comment:
why is the id being saved afterall? Can you confirm it's the `format` key being saved to the reducer?

Reply:
Oh sorry!